app/assets/javascripts/admin/addon/components/admin-flag-item.gjs

```diff
@@ -4,13 +4,22 @@ import { fn } from "@ember/helper";
 import { on } from "@ember/modifier";
 import { action } from "@ember/object";
 import { htmlSafe } from "@ember/template";
+import DButton from "discourse/components/d-button";
 import DToggleSwitch from "discourse/components/d-toggle-switch";
+import DropdownMenu from "discourse/components/dropdown-menu";
 import { ajax } from "discourse/lib/ajax";
 import { popupAjaxError } from "discourse/lib/ajax-error";
+import i18n from "discourse-common/helpers/i18n";
+import DMenu from "float-kit/components/d-menu";
+const NOTIFY_USER_ID = 6;
 
 export default class AdminFlagItem extends Component {
   @tracked enabled = this.args.flag.enabled;
 
+  get canMove() {
+    return this.args.flag.id !== NOTIFY_USER_ID;
```

Comment:
If you make the const change I suggested then you could do `this.args.flag.id !== SYSTEM_FLAG_IDS.notify_user`